### DISCUSSION TITLE
首页 · Pytorch 中文文档 - 交流板块

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/pytorch-doc-zh#593] [Announcements]
https://pytorch.apachecn.org/

**Comment by newhotter** [comment]
0评论终结者

**Comment by fengdshen** [comment]
1评论终结者

**Comment by SylvanLiu** [comment]
2评论终结者

**Comment by newhotter** [comment]
？？？。？



---原始邮件---
发件人: "刘思源"<notifications@github.com&gt;
发送时间: 2020年2月25日(周二) 凌晨5:45
收件人: "apachecn/pytorch-doc-zh"<pytorch-doc-zh@noreply.github.com&gt;;
抄送: "Comment"<comment@noreply.github.com&gt;;"newhotter"<10512076@qq.com&gt;;
主题: Re: [apachecn/pytorch-doc-zh] 首页 · Pytorch 中文文档 (#367)



2评论终结者

**Comment by ljflinux** [comment]
## 点开教程 -> 收藏网页 -> 关闭窗口，一气呵成。

**Comment by HarborZeng** [comment]
> @ljflinux
> ## 点开教程 -> 收藏网页 -> 关闭窗口，一气呵成。
> 

你太机智了

**Comment by Eric-Sjt** [comment]
请问从1.0开始没有EPUB等版本。如果要自己做应该怎么弄?

**Comment by liucong521** [comment]
怎么下载中文文档啊，怎么没法看啊？

**Comment by YoucanBaby** [comment]
感谢汉化

**Comment by ysh-strive** [comment]
点开教程 -> 收藏网页 -> 关闭窗口，一气呵成。

**Comment by FishCMD** [comment]
# 请问 __.whl__ 文件在哪里下？

**Comment by jiangzhonglian** [comment]
> # 请问 **.whl** 文件在哪里下？

我们只提供文档翻译，其他的可以去 http://pytorch.org 下载
